docs/presentations/2024-08-27-CU-Anschutz-DBMI-Retreat/readme.md

```diff
@@ -0,0 +1,45 @@
+# 2024-08-27 University of Colorado Anschutz Medical Campus - DBMI Retreat Poster
+
+The content here is for creating a poster for the University of Colorado Anschutz Medical Campus - Department of Biomedical Informatics (DBMI) Annual Retreat poster session.
+
+## Poster Details
+
+Poster dimensions will be within (but may not exactly match) SBI2 2023's maximum specifications: `91" wide x 44.75” high`.
```

Comment:
This is a big poster! Is there a reason you chose these dimensions specifically?

Reply:
Thanks - these are maximum constraints and not actual dimensions. For the actuals I used `48inx36in` (defined within the `qmd` frontmatter).